links/src/main/java/juja/microservices/links/model/Link.java

```diff
@@ -1,30 +1,31 @@
 package juja.microservices.links.model;
 
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
 import lombok.AllArgsConstructor;
-import lombok.EqualsAndHashCode;
-import lombok.Getter;
+import lombok.Data;
 import lombok.NoArgsConstructor;
-import lombok.Setter;
 import org.hibernate.validator.constraints.NotEmpty;
-import org.springframework.data.annotation.Id;
 
-@Getter
-@Setter
+/**
+ * @author Ivan Shapovalov
+ */
+@Data
 @AllArgsConstructor
-@EqualsAndHashCode
 @NoArgsConstructor
+@ApiModel
 public class Link {
-    @Id
+
+    @ApiModelProperty(value = "Id of saved link")
     private String id;
     @NotEmpty
+    @ApiModelProperty(value = "URL of saved link", required = true)
     private String url;
 
+    @ApiModelProperty(value = "Flag that means link is hidden or active")
```

Comment:
i'd remove final part "or active". if you want to use it as a substitute for "not hidden", there are a lot of more evident words like open, public, visible, apparent so on. but as for me its better to use a simple negation: "Flag that means the link is hidden or **not**".

Reply:
You're right